webapp/tests/karma/ts/services/db-sync.service.spec.ts

```diff
@@ -142,13 +150,13 @@ describe('DBSync service', () => {
       return service.sync().then(() => {
         expect(hasAuth.callCount).to.equal(1);
         expect(hasAuth.args[0][0]).to.equal('can_edit');
-        expect(from.callCount).to.equal(1);
+        expectSyncCall();
```

Comment:
Could you please pass in the number of calls expected instead of relying on the default value? This applies to all other calls relying on the default value. It makes it easier to read IMO.